examples/connext_dds/flat_data_latency/c++11/CameraImage_publisher.cxx

```diff
@@ -119,8 +119,12 @@ void publisher_flat(const ApplicationOptions &options)
             }
         }
     }
-    std::cout << "Average end-to-end latency: "
+    if (count) {
```

Comment:
`count > 0` for clarity.

Reply:
should we move it to a function? print_latency(total_latency, count) ?

Reply:
Updated the changes